api/src/main/java/com/sanction/lightning/models/FacebookUser.java

```diff
@@ -0,0 +1,104 @@
+package com.sanction.lightning.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+import com.restfb.Facebook;
+
+public class FacebookUser {
+
+  @Facebook
+  @JsonProperty("name")
+  private String name;
+
+  @Facebook
+  @JsonProperty("first_name")
```

Comment:
These `@JsonProperty` names should be camel case. e.g. `@JsonProperty("firstName")`